src/App.js

```diff
@@ -1,23 +1,37 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import Header from "./components/header/Header";
+import Main from "./components/main/Main";
+import Footer from "./components/footer/Footer";
+import data from "./data.json";
 
 function App() {
+  console.log(data);
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      {/* <Main
+        characterName="Harry Potter"
+        house="Gryffindor"
+        imgUrl="http://hp-api.herokuapp.com/images/harry.jpg"
+        actor="Daniel Radcliffe"
+      />
+      <Main
+        characterName="Hermione Granger"
+        house="Gryffindor"
+        imgUrl="http://hp-api.herokuapp.com/images/hermione.jpeg"
+        actor="Emma Watson"
+      /> */}
```

Comment:
Since you don't need this code anymore, please remove it.

Reply:
ich habe entfernt.